index.js

```diff
@@ -51,7 +51,7 @@ function attachComments(text, ast, opts) {
 function format(text, opts) {
   const ast = parse(text, opts);
   attachComments(text, ast, opts);
-  const doc = printAstToDoc(ast, opts);
+  const doc = printAstToDoc(ast, opts, text);
```

Comment:
`options.originalText` contains the raw text, you don't need to thread it through there.